airbyte-webapp/src/components/connectorBuilder/Builder/BuilderSidebar.tsx

```diff
@@ -66,97 +59,87 @@ interface BuilderSidebarProps {
   };
 
   return (
-    <div className={classnames(className, styles.container)}>
+    <FlexContainer direction="column" alignItems="stretch" gap="xl" className={classnames(className, styles.container)}>
       <UiYamlToggleButton yamlSelected={false} onClick={toggleYamlEditor} />
 
-      {/* TODO: replace with uploaded img when that functionality is added */}
-      <img
-        className={styles.connectorImg}
-        src="/logo.png"
-        alt={formatMessage({ id: "connectorBuilder.connectorImgAlt" })}
-      />
-
-      <div className={styles.connectorName}>
-        <Heading as="h2" size="sm" className={styles.connectorNameText}>
-          {values.global?.connectorName}
-        </Heading>
-      </div>
-
-      <ViewSelectButton
-        data-testid="navbutton-global"
-        className={styles.globalConfigButton}
-        selected={selectedView === "global"}
-        showErrorIndicator={hasErrors(true, ["global"])}
-        onClick={() => {
-          handleViewSelect("global");
-          analyticsService.track(Namespace.CONNECTOR_BUILDER, Action.GLOBAL_CONFIGURATION_SELECT, {
-            actionDescription: "Global Configuration view selected",
-          });
-        }}
-      >
-        <FontAwesomeIcon icon={faSliders} />
-        <FormattedMessage id="connectorBuilder.globalConfiguration" />
-      </ViewSelectButton>
-
-      <ViewSelectButton
-        data-testid="navbutton-inputs"
-        showErrorIndicator={false}
-        className={styles.globalConfigButton}
-        selected={selectedView === "inputs"}
-        onClick={() => {
-          handleViewSelect("inputs");
-          analyticsService.track(Namespace.CONNECTOR_BUILDER, Action.USER_INPUTS_SELECT, {
-            actionDescription: "User Inputs view selected",
-          });
-        }}
-      >
-        <FontAwesomeIcon icon={faUser} />
-        <FormattedMessage
-          id="connectorBuilder.userInputs"
-          values={{
-            number: values.inputs.length + getInferredInputs(values.global, values.inferredInputOverrides).length,
-          }}
+      <FlexContainer direction="column" alignItems="center">
+        {/* TODO: replace with uploaded img when that functionality is added */}
+        <img
+          className={styles.connectorImg}
+          src="/logo.png"
+          alt={formatMessage({ id: "connectorBuilder.connectorImgAlt" })}
         />
-      </ViewSelectButton>
 
-      <div className={styles.streamsHeader}>
-        <Text className={styles.streamsHeading} size="xs" bold>
-          <FormattedMessage id="connectorBuilder.streamsHeading" values={{ number: values.streams.length }} />
-        </Text>
-
-        <AddStreamButton onAddStream={(addedStreamNum) => handleViewSelect(addedStreamNum)} data-testid="add-stream" />
-      </div>
+        <div className={styles.connectorName}>
+          <Heading as="h2" size="sm">
+            {values.global?.connectorName}
+          </Heading>
+        </div>
+
+        <FlexItem>
```

Comment:
It doesn't seem like wrapping the `<SavingIndicator>` in a `<FlexItem>` is necessary here, since you aren't passing any props to the FlexItem